requirements.in

```diff
@@ -1,3 +1 @@
 -e file:.
-# DON'T FORGET TO UPDATE setup.py !!
```

Comment:
Do we still need to update [setup.py](https://github.com/alphagov/digitalmarketplace-content-loader/blob/master/setup.py)? It's looking rather outdated:

```
        'digitalmarketplace-utils>=52.9.0',
```

Reply:
We could, but I'd rather not... having to tight constraints on requirements in libraries can lead to problems. I think it's better to leave it up to consumers to to keep themselves updated, although we haven't been very good at that historically...

Reply:
Makes sense. Hopefully the pypy-ification will make this easier in future.